deployer/deployer-services/src/main/java/io/hyscale/deployer/services/deployer/Deployer.java

```diff
@@ -90,6 +91,9 @@ public default void waitForDeployment(DeploymentContext context, ProgressHandler
 	 */
 	public List<DeploymentStatus> getDeploymentStatus(DeploymentContext deploymentContext) throws HyscaleException;
 
+	
+	public List<ReplicaInfo> getReplicas(DeploymentContext deploymentContext, boolean isFilter) throws HyscaleException;
+	
```

Comment:
Deployment context is not required here right ? API should declare what all it required .